index.js

```diff
@@ -1,47 +1,124 @@
-const TelegramBot = require('node-telegram-bot-api');
-const fs = require('fs');
+const Telegraf = require('telegraf');
+const Extra = require('telegraf/extra');
 require('dotenv').config();
 
-let greeting = fs.readFileSync('./greeting.txt', { encoding: 'utf-8' });
-
-const bot = new TelegramBot(process.env.BOT_TOKEN, {
-    polling: true,
-    timeout: 500
-});
+const bot = new Telegraf(process.env.BOT_TOKEN);
 
-bot.on('message', msg =>{
-    if (msg.new_chat_members) {
-        const new_user = msg.new_chat_member;
-
-        new_user.username = `${new_user.first_name || ''} ${new_user.last_name || ''}`
+const fs = require('fs');
```

Comment:
all imports (requires)of packages/modules should place on top of the file
just one of "best practices" in JS